_clients/data-prepper/data-prepper-reference.md

```diff
@@ -92,6 +92,42 @@ awsRegion | Conditionally | String | Represents the AWS Region to use ACM or S3.
 authentication | No | Object | An authentication configuration. By default, an unauthenticated server is created for the pipeline. This uses pluggable authentication for HTTPS. To use basic authentication, define the `http_basic` plugin with a `username` and `password`. To provide customer authentication, use or create a plugin that implements [GrpcAuthenticationProvider](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/armeria-common/src/main/java/com/amazon/dataprepper/armeria/authentication/GrpcAuthenticationProvider.java).
 
 
+### s3
+
+This is a source plugin that reads events from S3 objects.
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+notification_type | Yes | String | Must be `sqs`
+compression | No | String | The compression algorithm to apply; `none`, `gzip`, or `automatic`. Default is `none`.
+codec | Yes | Codec | The codec to apply. Must be either `newline` or `json`.
+sqs | Yes | sqs | The SQS configuration. See [sqs](#s3-source-sqs) for details.
+aws | Yes | aws | The AWS configuration. See [aws](#s3-source-aws) for details.
+on_error | No | String |  Determines how to handle errors in SQS. Can be either `retain_messages` or `delete_messages`. If `retain_messages`, then Data Prepper will leave the message in the SQS queue and try again. This is recommended for dead-letter queues. If `delete_messages`, then Data Prepper will delete failed messages. Default is `retain_messages`.
+buffer_timeout | No | Duration | The timeout for writing events to the Data Prepper buffer. Any events which the S3 Source cannot write to the buffer in this time will be discarded. Default is 10 seconds.
+records_to_accumulate | No | Integer | The number of messages that accumulate before writing to the buffer. Default is 100.
+disable_bucket_ownership_validation | No | Boolean | If `true`, then the S3 Source will not attempt to validate that the bucket is owned by the expected account. The only expected account is the same account which owns the SQS queue. Defaults to `false`.
+
+#### <a name="s3-source-sqs">sqs</a>
+
+SQS configurations for the S3 source plugin
```

Comment:
Change "SQS" to "Amazon SQS". Capitalize "Source". It also looks like this should be rewritten as a complete sentence.